pycontw/pycon.tw
# Remove unused variables for using DJ-Registry

The commit 4c1a1fc400d00ad4be963c902d9a92325804d08e introduced the usage
of DJ-Registry so we do not need the corresponding variables in the
configuration file. This commit follows up the the introducing of
DJ-Registry and perform more comprehensive house cleaning.

# How to Test This Pull Request
1. Get ready to have the permission to access the Django administration of the development site e.g. your local site.
1. Create at least two users and setup one of them as a reviewer.
1. Follow the README of the review system. Change the variables for each review stage[1].
1. Check the `My PyCon` dashboard behavior is the same as what the README describes.

[1] A screenshot of my development Django administration console

![Selection_166](https://user-images.githubusercontent.com/3217432/82069711-4ddabc00-9706-11ea-92b2-7d012c41aedf.png)
